### DISCUSSION TITLE
【GitHub Sponsors】How do I delete my stripe express account

### DISCUSSION THREAD

**Discussion by maomincoding** [community/community#145600] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I accidentally created a github sponsors account, which was created based on Stripe Express. So I asked Stripe officials, and they said that it needs to be deleted on the platform where it was created. Then I filed a ticket with Github, but it has been three days and there has been no response. I have also filed many tickets before, but there has been no response. Has anyone encountered this situation?

**Comment by maomincoding** [comment]
I did not find any button or entry to delete or log out of an account at [https://connect.stripe.com/app/express]( https://connect.stripe.com/app/express)

**Comment by maomincoding** [comment]
It seems that I have to contact GitHub Sponsors to help me delete my account? It seems that filing a ticket is useless. I don't know what to do next.

**Comment by maomincoding** [comment]
@DHayes10

**Reply by samus-aran** [reply]
Although you wish to have an immediate response please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by samus-aran** [comment]
Hi @maomincoding 👋 
The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

Same goes for discussions. We can see multiple discussions so we will close this one in favour for this other one: https://github.com/orgs/community/discussions/144897

**Comment by maomincoding** [comment]
I’m very sorry, because I‘m in a hurry. I want to delete my stripe Express account.

---- Replied Message ----
| From | Ashley ***@***.***> |
| Date | 11/29/2024 19:03 |
| To | community/community ***@***.***> |
| Cc | Vam ***@***.***>,
Mention ***@***.***> |
| Subject | Re: [community/community] 【GitHub Sponsors】How do I delete my stripe express account (Discussion #145600) |

Hi @maomincoding 👋
The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

Same goes for discussions. We can see multiple discussions so we will close this one in favour for this other one: https://github.com/orgs/community/discussions/144897

**Comment by maomincoding** [comment]
Sorry, it was my negligence. I apologize again for my reckless behavior. Could you please help me delete the Stripe Express account associated with Github Sponsor? Because I didn't understand the consequences of registration before, I can't use this account now, so I want to delete it. Thank you！











在 2024-11-29 19:03:14，"Ashley Nicolson" ***@***.***> 写道：

Hi @maomincoding
The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

Same goes for discussions. We can see multiple discussions so we will close this one in favour for this other one: https://github.com/orgs/community/discussions/144897
